Rtsp/RtspClient.cs

```diff
@@ -2602,8 +2616,7 @@ internal void ProcessInterleavedData(object sender, byte[] data, int offset, int
         /// <summary>
         /// Increments and returns the current <see cref="ClientSequenceNumber"/>
         /// </summary>
-        [System.Runtime.CompilerServices.MethodImpl(System.Runtime.CompilerServices.MethodImplOptions.Synchronized)]
-        internal int NextClientSequenceNumber() { return ++m_CSeq; }
+        internal int NextClientSequenceNumber() { return Interlocked.Increment(ref m_CSeq); }
```

Comment:
on x86 / x64 writes which fit in a register are already atomic. We can keep the interlocked though as its more appropirate.